packages/api/lib/granules.js

```diff
@@ -33,14 +34,21 @@ const translateGranule = async (
   granule,
```

Comment:
Not a problem *from* this PR, but there's no unit around this method directly.    This logic needs a doc and unit test.

Reply:
added doc and unit test